src/decision/location_selection.jl

```diff
@@ -124,7 +125,14 @@ function rank_locations(
         depth_priority = findall(depth_criteria)
 
         considered_sites = target_site_ids[findall(in(depth_priority), target_site_ids)]
-        mcda_vars_temp = DMCDA_vars(domain, scen, considered_sites,  sum_cover[scen_idx, :], area_to_seed,
+        mcda_vars_temp = DMCDA_vars(
+            domain,
+            scen,
+            considered_sites,
+            relative_leftover_space(
+                AxisKeys.keyless(NamedDims.unname(sum_cover[scen_idx, :]))
```

Comment:
> Sorry, its not the argument to rank_locations which is wrong, it's the argument to MCDA_vars in rank_locations, which now takes leftover_space, rather than cover. rank_locations still takes cover. The update to MCDA_vars was done when we added a depth weight, I believe. I'll make this clearer in my description.

Sorry, I'm still not following - but I think what you're raising is a separate issue.

If `sum_cover` is in absolute units (e.g., meters squared) then this change to L 132 of `location_selection.jl` 
is wrong because `relative_leftover_space` expects relative cover.

https://github.com/open-AIMS/ADRIA.jl/blob/62c5deb4505a424233f2c724117767ae27a059ba/src/Domain.jl#L234-L249

Reply:
I think you've spotted another issue, the input to `_location_selection` should now be relative cover because it is use in `connectivity_strength`, so the input to `rank_locations` should also be relative cover ( and was originally I think), I'll add changes to the docs

Reply:
>  the input to _location_selection should now be relative cover because it is used in connectivity_strength, so the input to rank_locations should also be relative cover

`connectivity_strength` uses absolute cover. I don't think it should use relative cover, wouldn't create misleading indications of connectivity, as it places greater weight on small locations with high cover?

The transference probability matrix  (TP) is expected to be weighted by $k$ area based on the docstring.

I see yet another issue here - the process of loading the domain for ADRIA uses the TP data directly, whereas loading ReefMod Engine datasets applies the weighting based on k area and absolute cover.

Reply:
Right, I just copied the usage in `scenario.jl`, whith uses `TP` weighted with `k_area` for the first argument and relative cover for the second. Should the second argument be absolute cover then?

Reply:
Okay, could you please check which it should actually be then?

Personally I think my previous change in `scenario.jl` is incorrect.

Reply:
Sure, I'll check :)

Reply:
I think relative cover in `connectivity_strength` is correct because here it is multiplied byt the k_area weighted TP matrix- https://github.com/open-AIMS/ADRIA.jl/blob/62c5deb4505a424233f2c724117767ae27a059ba/src/ecosystem/connectivity.jl#L218

Reply:
Okay, so it should be relative cover and the previous change in `scenario.jl` is correct? In which case the docstring needs to be updated

Reply:
Ok cool, I'll update the doc strings, then I think it's ready for review @Zapiano